conda/recipes/cudf_kafka/conda_build_config.yaml

```diff
@@ -6,3 +6,6 @@ cxx_compiler_version:
 
 sysroot_version:
   - "2.17"
+
+cmake_version:
```

Comment:
I tend to avoid using `conda_build_config.yaml` for defining a variable when the version is only specified once in a recipe. I see the value of being consistent between `cudf_kafka` and other recipes, though.

Reply:
Yeah I made this change since it then made it easily to do a batch upgrade across RAPIDS when everything is consistent. Could always add one more file to sed though.